metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -124,6 +124,14 @@ class KafkaSourceConfig(StatefulIngestionConfigBase, DatasetSourceConfigMixin):
         default=False,
         description="Disables the utilization of the TopicRecordNameStrategy for Schema Registry subjects. For more information, visit: https://docs.confluent.io/platform/current/schema-registry/serdes-develop/index.html#handling-differences-between-preregistered-and-client-derived-schemas:~:text=io.confluent.kafka.serializers.subject.TopicRecordNameStrategy",
     )
+    registry_name: str = pydantic.Field(
+        default={},
+        description="The name of the Glue Schema Registry from where the schema metadata will be ingested.",
+    )
+    aws_account_region: str = pydantic.Field(
+        default={},
+        description="The name of the Glue Schema Registry from where the schema metadata will be ingested.",
```

Comment:
Description needs to be updated here. 

Can we add a class `GlueSchemaRegistryOptions` and define these fields inside the class.
such that kafka config has a field:
`glue_schema_registry_options: Optional[GlueSchemaRegistryOptions] # Required if using Glue Schema Registry`